core/camel-endpointdsl/src/generated/java/org/apache/camel/builder/endpoint/EndpointBuilderFactory.java

```diff
@@ -28,14 +28,12 @@
 @Generated("org.apache.camel.maven.packaging.EndpointDslMojo")
 public interface EndpointBuilderFactory
         extends
-            org.apache.camel.builder.endpoint.dsl.AMQPEndpointBuilderFactory.AMQPBuilders,
-            org.apache.camel.builder.endpoint.dsl.AS2EndpointBuilderFactory.AS2Builders,
-            org.apache.camel.builder.endpoint.dsl.AWS2EC2EndpointBuilderFactory.AWS2EC2Builders,
-            org.apache.camel.builder.endpoint.dsl.AWS2S3EndpointBuilderFactory.AWS2S3Builders,
             org.apache.camel.builder.endpoint.dsl.ActiveMQEndpointBuilderFactory.ActiveMQBuilders,
             org.apache.camel.builder.endpoint.dsl.AhcEndpointBuilderFactory.AhcBuilders,
+            org.apache.camel.builder.endpoint.dsl.AMQPEndpointBuilderFactory.AMQPBuilders,
```

Comment:
Here I guess you have a mix of empty folders or a git tree not clean.

Reply:
Strange. I cleaned up my local branches and reverted the changes on EndpointBuilderFactory and EndpointBuilders file. Let me do a check-in with reverted state.

Reply:
> Here I guess you have a mix of empty folders or a git tree not clean.

I did a revert, complete cleanup and applied changes and did  a mvn build. I see the same set of changes again. Could it be extensions getting re-ordered inside the same file? I see ```org.apache.camel.builder.endpoint.dsl.AMQPEndpointBuilderFactory.AMQPBuilders``` removed from upper portion of the file and moved down ?

![image](https://user-images.githubusercontent.com/81697586/114240094-f7e1b900-9954-11eb-9185-a9abe2f18af4.png)

Reply:
It could be. The important point is not losing builders